install/kubernetes/webhook-create-signed-cert.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    cat <<EOF
+Generate certificate suitable for use with an Istio webhook service.
```

Comment:
pretty sure I saw this code elsewhere before... and/or we have stuff checked in that makes it no needed?
@ajy @wattli

Reply:
We no longer need this script for injector webhook so I removed it in #4612 . However, there is an e2e test for admin webhook for mixer validator. So I have to restore it temporarily to fix the tests.

https://github.com/istio/istio/blob/master/tests/e2e/framework/kubernetes.go#L196